### PR TITLE
[plugin.video.dumpert] 1.1.10+matrix.1

### PR DESCRIPTION
**Pull request by skipmodea1**
### Description
Made a python 3 version based upon my python 2 version.
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.dumpert/addon.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">Watch funny videos from Dumpert.nl (dutch)</summary>
+        <description lang="en_GB">Watch funny videos from Dumpert.nl (dutch)</description>
+        <disclaimer lang="en_GB">For bugs, requests or general questions visit the Dumpert.nl thread on the Kodi forum.</disclaimer>
+    <summary lang="nl_NL">Bekijk grappige filmpjes van Dumpert.nl (dutch)</summary>
+        <description lang="nl_NL">Bekijk grappige filmpjes van Dumpert.nl (dutch)</description>
+        <disclaimer lang="nl_NL">Bugs of andere feedback op deze plugin kan geplaatst worden in de Dumpert.nl thread op het Kodi forum.</disclaimer>
+    <language>nl</language>
+    <platform>all</platform>
+    <license>GPL-2.0-or-later</license>
```
Comment (basrieter) [inline plugin.video.dumpert/addon.xml]: The same issue with the license here. It should be "GPL-3.0" as your LICENSE.txt has "GNU GENERAL PUBLIC LICENSE 3, 29 June 2007"

**Review by basrieter — CHANGES_REQUESTED** [review]
Please fix the license in addon.xml. It should be the same type as the LICENSE.txt and the non-Matrix version of the Dumpert Video Add-on.

**Comment by enen92** [comment]
@basrieter also noticed this but gpl2.0 or later includes gpl3 I'd say, so decided to ignore

**Comment by basrieter** [comment]
> @basrieter also noticed this but gpl2.0 or later includes gpl3 I'd say, so decided to ignore

It should at least be consistent for the Matrix and non-Matrix versions.

File: plugin.video.dumpert/addon.xml (hunk 1 above)
Comment (skipmodea1) [reply plugin.video.dumpert/addon.xml]: Fixed

**Review by basrieter — APPROVED** [review]

**Pull request merged**